doc/src/modules/assumptions/index.rst

```diff
@@ -32,7 +32,7 @@ assumptions about objects in expr. Valid values include:
     * Q.integer(x) & Q.positive(x)
     * etc.
 
-Q is a class in sympy.assumptions holding known predicates.
+Q is an instance of QClass in sympy.assumptions.ask holding known predicates.
```

Comment:
Just "is an object holding known predicates".
Unless `QClass` is supposed to be part of the official SymPy API, that is.
(I suspect the class is misnamed, too, but that's beyond this PR's scope.)